src/svelte/App.svelte

```diff
@@ -56,7 +56,6 @@
   {:else if todoGroups.length === 0}
     <div>No checklist items found for tag: #{todoTag}</div>
   {:else}
-    <div class="header" />
```

Comment:
Didn't see this being used anywhere.

Reply:
Yep, I never ended up using that. Good catch